source/images/fellowships/svg/support-fellows-optim.svg

```diff
@@ -1,3 +1,3 @@
1 25.8 41.8 34.4l1 .7.3-.2.3.2 1-.7c12.7-8.6 27.8-20.3 41.7-34.3l3.8 3.8 1.3-1.3-3.8-3.9c10.4-10.7 20-22.8 27.1-36 10.2-18.9 10.7-31.1 10.7-31.6l.1-2.2h-.1c.1-.6.1-1.2.1-1.8zm-3.9 3.8l-.2 2.4h.1c-.6 4.3-2.7 14.3-10.1 27.9-7 12.7-16.2 24.4-26.3 34.8l-4.1-4.1-1.3 1.3 4.1 4.1c-13.3 13.5-27.8 24.7-40.1 33.1l-.3-.2-.3.2c-12.3-8.5-26.9-19.7-40.2-33.2l4-4-1.3-1.3-4 4c-10-10.4-19.3-22-26.2-34.7-1.3-2.3-2.4-4.5-3.3-6.6h3.7v-1.9h-3.8v1.8C14 67.4 12.4 60 11.9 56.4h.1l-.1-2.4c-.1-1.2-.2-2.5-.2-3.8 0-13.5 5.6-25.7 14.4-34.3l2.1 2.1 1.3-1.3-2-2c7.7-6.8 17.7-10.9 28.6-11 12.3 0 24.2 5.4 32.6 14.9l1.4 1.6 1.4-1.6c8.4-9.5 20.3-14.9 32.6-14.9 10.8 0 20.7 4.1 28.5 10.8l-2 2 1.3 1.3 2.1-2.1c8.9 8.5 14.5 20.8 14.5 34.4 0 1.4-.1 2.6-.2 3.9zM40.2 30L29.6 19.4l1.3-1.3 10.6 10.6-1.3 1.3zm1.3 1.3l1.3-1.3 10.6 10.6-1.3 1.4-10.6-10.7zm12 12l1.3-1.3 1.7 1.7-1.3 1.3-1.7-1.7zm83.8-13.4l1.3 1.3L128 41.9l-1.3-1.3 10.6-10.7zm-12 12l1.3 1.3-1.7 1.7-1.3-1.3 1.7-1.7zm14.6-12l-1.3-1.3L149.2 18l1.3 1.3-10.6 10.6zm-84.5 82l1.3 1.3-1.7 1.7-1.3-1.3 1.7-1.7zm-15 15l1.3 1.3-10.6 10.6-1.3-1.3 10.6-10.6zm-11.9 11.9l1.3 1.3-10.6 10.6-1.3-1.3 10.6-10.6zm96.2-26.9l1.7 1.7-1.3 1.3-1.7-1.7 1.3-1.3zm26.9 26.9l10.6 10.6-1.3 1.3-10.6-10.6 1.3-1.3zm-1.4-1.3l-1.3 1.3-10.6-10.6 1.3-1.3 10.6 10.6zm-28.3-40.1c-1.2-7.3-3.5-13-6.6-16.9-3.3-4.1-7.3-4.8-10.2-5.4-.7-.1-1.4-.3-2.1-.4-3.3-.9-5.3-2.7-5.3-4.6V70c0-1 0-1.7.1-2.1 5.6-3.8 8.5-9.9 8.9-18.6.3-6.8-1.6-12.7-5.3-16.5-2.9-3-6.8-4.6-11.2-4.6h-.3c-4.4 0-8.2 1.5-11.1 4.5-3.7 3.8-5.6 9.7-5.3 16.6.3 8.7 3.2 14.8 8.9 18.6.1.4.1 1.1.1 2.1v.1c0 1.9-2 3.6-5.3 4.6-.6.2-1.3.3-2.1.4-2.9.6-6.9 1.3-10.2 5.4-3.2 3.9-5.4 9.5-6.6 16.9v.1c-.1.8.1 2.1 1.2 3.3 5.3 5.9 30.2 6.1 30.4 6.1h.4c.2 0 25.1-.1 30.4-6.1 1.1-1.2 1.3-2.5 1.2-3.3v-.1zm-3.9 1c-2.8 3.1-17.8 4.8-27.6 4.8H90c-9.8 0-24.8-1.7-27.6-4.8-.2-.2-.2-.3-.2-.4 1.1-6.6 3.1-11.7 5.8-15 2.5-3 5.3-3.5 8-4.1.8-.2 1.6-.3 2.4-.5 5-1.4 8-4.5 8-8.2v-.1c0-2-.1-4-1.5-5 0 0-.1 0-.1-.1-4.7-3.1-7.2-8.3-7.5-15.8-.2-5.8 1.3-10.7 4.3-13.8 2.2-2.3 5-3.4 8.4-3.4h.4c3.4 0 6.2 1.1 8.4 3.4 3 3.1 4.5 8 4.3 13.8-.3 7.6-2.7 12.8-7.5 15.8 0 0-.1 0-.1.1-1.3 1-1.5 3-1.5 5v.1c0 3.7 3 6.8 8 8.2.8.2 1.5.4 2.4.5 2.7.5 5.5 1.1 8 4.1 2.7 3.3 4.7 8.4 5.8 15 0 0 0 .2-.2.4zm-72.5-.7h3.2v3.8h-3.2v3.3h-3.8v-3.3h-3.2v-3.8h3.2v-3.3h3.8v3.3zm8.7-15.2h3.2v3.8h-3.2v3.3h-3.8v-3.3h-3.2v-3.8h3.2v-3.3h3.8v3.3zm69.5-65.4c-.9.5-1.2.8-1.7 1.7l-.8 1.6-.8-1.6c-.5-.9-.8-1.2-1.7-1.7l-1.6-.8 1.6-.8c.9-.5 1.2-.8 1.7-1.7l.8-1.6.8 1.6c.5.9.8 1.2 1.7 1.7l1.6.8-1.6.8zM112 32.5l-2.2-1.1 2.2-1.1c1.3-.6 1.7-1.1 2.3-2.4l1.1-2.2 1.1 2.2c.6 1.3 1.1 1.7 2.4 2.3l2.2 1.1-2.2 1.1c-1.3.6-1.7 1.1-2.3 2.4l-1.1 2.2-1.1-2.2c-.7-1.2-1.1-1.7-2.4-2.3zM9.4 75.7h3.8v1.9H9.4v-1.9zm18.7 0h3.8v1.9h-3.8v-1.9zm13.2 1.9h-3.8v-1.9h3.8v1.9zM0 75.7h3.8v1.9H0v-1.9zm89.2 87.5h1.9v3.8h-1.9v-3.8zm0-37.5h1.9v3.8h-1.9v-3.8zm0 18.8h1.9v3.8h-1.9v-3.8zm0-9.4h1.9v3.8h-1.9v-3.8zm58.9-59.4h3.8v1.9h-3.8v-1.9zm13.1 1.9h-3.8v-1.9h3.8v1.9zm-22.5-1.9h3.8v1.9h-3.8v-1.9zm28.2 0h3.8v1.9h-3.8v-1.9zm13.1 0v1.9h-3.8v-1.9h3.8z"/>
```

Comment:
Don't forget to optimize SVG files before you commit them! 👍 

You can do it by running [`svgo`](https://github.com/svg/svgo): `$> svgo path_to_file`

Reply:
I didn't know—sorry, I'll do it now! :)

Reply:
This file appears optimized already, so I'm going to skip this step for this one. But, I'll remember to do it for upcoming svgs.